jerry-core/jerry-port.h

```diff
@@ -139,6 +139,13 @@ bool jerry_port_get_time_zone (jerry_time_zone_t *tz_p);
  */
 double jerry_port_get_current_time (void);
 
+#ifndef CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
+#include "ecma-jobqueue.h"
```

Comment:
I think including this is not a good idea. IMHO we shouldn't make internal parts public. It would be great to separate the private and public parts more clearly.